src/amber/cli/templates/auth.cr

```diff
@@ -27,6 +27,9 @@ module Amber::CLI
       @visible_fields = @fields.reject(&.hidden).map(&.name)
 
       add_routes :web, <<-ROUTES
+        get "/profile", #{class_name}Controller, :show
+        get "/profile/edit", #{class_name}Controller, :edit
+        patch "/profile", #{class_name}Controller, :update
```

Comment:
This is where we might be getting scope creep - having a few profile actions.

Reply:
no, no.  this is great!  I have been wanting to add something like this.  Also, I have been wanting to do `forget password` and generate a mailer.

Reply:
Is the existing mailer example a good starting point? That should probably be another PR (forgot password functionality).

I'm assuming `SecureRandom.urlsafe_base64(16)` will be sufficient for generating a token.

Reply:
yeah, the `forget password` would be a different PR.